plugins/ps/functions

```diff
@@ -21,17 +21,21 @@ generate_scale_file() {
     trap 'rm -rf "$TMP_WORK_DIR" > /dev/null' RETURN
 
     copy_from_image "$IMAGE" "/app/Procfile" "$TMP_WORK_DIR" 2>/dev/null || true
+    copy_from_image "$IMAGE" "/opt/Procfile.compose" "$TMP_WORK_DIR" 2>/dev/null || true
```

Comment:
We generate that file. It's a mapping of `docker-compose.yml` processes to the correct docker image.

It's a really shitty hack and I'm not proud of it, but can't think of any alternatives at this time.